levels/fog_owl/maps/default.json

```diff
@@ -1006,6 +1006,10 @@
         {
          "firstgid":10795,
          "source":"..\/..\/..\/..\/.bundled\/tilesets\/TwilioQuestObjects.json"
+        }, 
+        {
+         "firstgid":11811,
+         "source":"..\/..\/..\/tilesets\/TwilioQuestObjects.json"
```

Comment:
The Fog Owl should not be referencing the direct TwilioQuestObjects tileset. It should only reference the bundled version above.

Reply:
@kerrielake This tileset still seems to be referencing the non-bundled TwilioQuestObjects tileset. The tileset above is correctly referencing the .bundled version.

Reply:
> @kerrielake This tileset still seems to be referencing the non-bundled TwilioQuestObjects tileset. The tileset above is correctly referencing the .bundled version.

Ok I think I discovered what happened. I was testing out the new object additions on the map, and it must have continued to reference the non-bundled version even after I removed them. I've updated the directory to .bundled in the json file, let me know if that has fixed the issue!